core/src/syn/parser/test/stmt.rs

```diff
@@ -1024,6 +1024,7 @@ fn parse_define_access_record() {
 				kind: AccessType::Record(RecordAccess {
 					signup: None,
 					signin: None,
+					authenticate: None,
```

Comment:
Other than updating the current tests. I will add a new tests here to ensure that the clause is parsed correctly.

Reply:
Done in [64c6e46](https://github.com/surrealdb/surrealdb/pull/4161/commits/64c6e465f26dd7824625d724387c43689d76ec34).